snapfaas/src/syscalls.proto

```diff
@@ -162,6 +159,7 @@ message Syscall {
     BlobOpen openBlob = 16;
     BlobRead readBlob = 17;
     BlobClose closeBlob = 18;
-    ReadDir readDir = 19;
+    Component declassify = 19;
+    ReadDir readDir = 20;
```

Comment:
You should (almost) never re-order protocol buffer field numbers (part of the point is that the field numbers make future protocol buffers backwards compatible). So definitely if it's not really necessary.